package.json

```diff
@@ -7,15 +7,7 @@
     "node": ">=11.0.0"
   },
   "main": "./dist/index.js",
-  "module": "./dist/index.mjs",
   "types": "./dist/index.d.ts",
-  "exports": {
-    ".": {
-      "import": "./dist/index.mjs",
-      "require": "./dist/index.js",
-      "types": "./dist/index.d.ts"
-    }
-  },
   "files": [
     "dist/index.d.ts",
     "dist/index.mjs",
```

Comment:
then we don't need to include separate mjs file anymore)

and later we might switch that to just including mjs and not .js, but that would require an audit of the diff